paper/paper.md

```diff
@@ -61,55 +77,89 @@ nf-core community [@ewels_nf-core_2020].
-building block in metagenomic analysis workflows. At the time of writing, it was
-able to read profiles from nine different profilers, namely Bracken
-[@lu_bracken_2017], Centrifuge [@kim_centrifuge_2016], DIAMOND
-[@buchfink_sensitive_2021], Kaiju [@menzel_fast_2016], Kraken2
-[@wood_improved_2019], KrakenUniq [@breitwieser_krakenuniq_2018], MEGAN6/MALT
-[@gautam_meganserver_2023], MetaPhlAn [@blanco-miguez_extending_2023], and mOTUs
-[@ruscheweyh_cultivation-independent_2022]. It offers a wide range of output
-file formats, such as text-based, tabular formats (CSV[^1], TSV[^2]), spreadsheets
-(ODS[^3], XLSX[^4]), optimized binary formats (Apache Arrow[^5] and
-Parquet[^6]), and the HDF5-based[^7] BIOM format [@mcdonald_biological_2012].
+information. Taxonomic identifiers can be recorded with taxonmic identifiers,
+taxonomic names and/or in some cases semi-colon-separated taxonomic 'paths'
+(lineages). These can also be formatted in different ways, from typical tables,
+to including 'indented' taxonomy trees such as in the Kraken
+[@wood_improved_2019] family of profilers. Manually parsing these for comparison
+can be an arduous, error-prone task, with researchers often reverting to custom
+R and Python scripting, or even manual correction in spreadsheet software.
 
-[^1]: https://en.wikipedia.org/wiki/Comma-separated_values
-[^2]: https://en.wikipedia.org/wiki/Tab-separated_values
-[^3]: https://en.wikipedia.org/wiki/OpenDocument
-[^4]: https://en.wikipedia.org/wiki/Office_Open_XML
-[^5]: https://arrow.apache.org/
-[^6]: https://parquet.apache.org/
-[^7]: https://www.hdfgroup.org/solutions/hdf5/
-
-TAXPASTA is not the first tool to attempt to standardise metagenomic profiles,
+With TAXPASTA, all of those formats can be converted into a single, standardised
+output, that, at a minimum, contains taxonomic identifiers and their relative
+abundances as integer counts. It can also be used to aggregate profiles across
+samples from the same profiler and merge them into a single, standardised table.
+Having a singular format facilitates downstream analyses and comparisons.
+TAXPASTA is not the first tool to attempt standardising metagenomic profiles,
 but it is by far the most comprehensive in terms of supported profilers and
 output formats.
 
+There exists an initiative to benchmark and compare profilers, as well as
+provide guidance on their fitness for purpose
+[@sczyrba_critical_2017; @meyer_critical_2022]. For that purpose, the
+Open-community Profiling Assessment tooL (OPAL) [@meyer_assessing_2019] was
+developed. Creating a community wide assessment faced many of the challenges
+presented here, however, the chosen solution was to mandate a single output
+format[^8] for all profilers participating in the challenge. Furthermore, OPAL
+is an integrated tool performing assessment and visualization, whereas TAXPASTA
+follows the UNIX philosophy[^9] of doing one thing well, thus allowing for
+composition with other tools.
```

Comment:
> What do you mean by composition with other tools? I thought the idea of that part of the UNIX philosophy was about piping?

That was my thinking, too. Composing analysis workflows by combining different tools. I'm happy to use different wording.

Reply:
> Note I wouldn't have a line break between the OPAL and BIOM formats, as it's the same 'section' of the paper.

You don't think it's a separate paragraph?

Reply:
> That was my thinking, too. Composing analysis workflows by combining different tools. I'm happy to use different wording.

It depends what you mean with composition with other tools. We do follow the unix philosphy of do one thing well, but we don't pipe so I would remove the `thus <...> other tools` bit

> You don't think it's a separate paragraph?

Without subheaders (which aren't really wanted in JOSS papers AFAICS), to me that's the section were we compare with other similar tools/formats (biom has it's own CLI IIRC). But this is me being nit picky, fine if you prefer separate.

Reply:
@jfy133 do you think that we **should** allow for streaming output such that piping becomes possible? It would be fairly straight forward to allow for this.

Reply:
I guess you could for stdout piping although I don't really see the utility much unless the output is in long format (wide you would lose the header from the output from merge)... And even then I can't really think of when you would use it in an analysis workflow...

I can't see how you would implement it for stdin though as you wouldn't have the sample ID info that in most cases is stored in the file name

But if it would take you 20 minutes I guess why not (??) But any longer would not be worth it

Reply:
I have adjusted the sentence for now, but it's definitely something we can consider.